content/documentation/validations/index.adoc

```diff
@@ -87,6 +87,9 @@ This validation warning could be a result of duplicate definition of the same su
 
 Istio silently ignores the duplicate subsets and merge these destination rules without letting the user know. Only the first seen rule (by Istio) per subset is used and information from multiple definitions is not merged. While the routing might work correctly, this is most likely a configuration error. It may lead to a undesired behavior if one of the offending rules is removed or modified and that is probably not the intention of the deployer of this service. Also, if the two offending destination rules have different policies for traffic routing the wrong one might be used.
 
+icon:exclamation-triangle[] Istio highly recommend to always use FQDN for hosts on DestinationRules. However it allows to use other formarts like short names ('details' or 'details.bookinfo').
```

Comment:
Istio highly recommends that you always use fully qualified domain names (FQDN) for hosts in DestinationRules.  However, Istio does allow you to use other formats like short names ('details' or 'details.bookinfo').